content/ja/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -120,9 +120,9 @@ Deploymentによって作成されたReplicaSetを管理しないでください
   コマンドの実行結果は以下のとおりです:
   ```shell
```

Comment:
The pod-template output changes are aligned with the 'en' PR

nit: Remove the unnecessary code type `shell` from all the output blocks in this file  _(Equivalent 'en' PR   #23270)_

Reply:
With this commit, I synchronize what is in the English version, the rest in another PR

Reply:
With this commit, I synchronize what is in the English version, the rest in another PR